tests/openqa/install/openqa_bootstrap_container.pm

```diff
@@ -0,0 +1,37 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018-2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Install openQA using openqa-bootstrap-container
+# Maintainer: Dominik Heidler <dheidler@suse.de>
+
+use strict;
+use base "consoletest";
+use testapi;
+use utils;
+
+
+sub run {
+    select_console 'root-console';
+
+    if (script_run('stat /dev/kvm') != 0) {
+        record_info('No nested virt', 'Creating dummy /dev/kvm');
+        assert_script_run('mknod /dev/kvm c 10 232');
+    }
```

Comment:
[DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself). You are calling the same code multiple times, please [extract method](http://refactoring.com/catalog/extractMethod.html) or use a base class.

Reply:
It is actually only really useful for the container test (becaus the container doesn't start without this as it is referenced from container config) so I will just remove the code from openqa_bootstrap.pm.